index.js

```diff
@@ -33,6 +35,23 @@ module.exports = async function standardVersion (argv) {
     }
   }
 
+  if (argv.detectRelease) {
```

Comment:
I would be tempted to pull this into a `detectRelease` helper method.

Reply:
Just to confirm, are you suggesting to move the logic we have inside this `if` to it's own file and expose a `detectRelease` method that we could use call here?

Reply:
@nelsonfncosta yes, I think it would make sense in its own helper method.